batch2/deployment.yaml

```diff
@@ -261,3 +261,13 @@ spec:
      resource:
        name: cpu
        targetAverageUtilization: 80
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: batch2
```

Comment:
Do you need a second one for batch2-driver? I assume not because it's ok if the driver is offline.

Reply:
Not sure about the previous batch2-driver, there were some issues about long startup.  I was planning to make the driver tolerate preemptibles (with a min 0 pdb) after my remove pods PR goes in.  Le'ts wait for that.